app/controllers/home_controller.rb

```diff
@@ -1,15 +1,11 @@
 class HomeController < ApplicationController
   def home
-    if(params[:re_login])
-      redirect_to user_omniauth_authorize_path(params[:provider])
```

Comment:
This feature can still be partially supported by using the "domain" parameter on any route in LARA.
One difference though, is the redirecting.  The code removed here leaves the user on the referrer page. So if this is code is run after a series of redirects then the user will end up back where they started. The "domain" parameter returns the user to same URL without the domain parameter, not the referrer.

As far as I can tell this re_login was only used by other code removed in this PR, so I'm not worried about removing it.